src/main/java/com/twilio/oai/api/CsharpApiResources.java

```diff
@@ -0,0 +1,19 @@
+package com.twilio.oai.api;
+
+import com.twilio.oai.common.ApplicationConstants;
+import com.twilio.oai.resolver.csharp.OperationStore;
+import org.openapitools.codegen.IJsonSchemaValidationProperties;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsharpApiResources extends ApiResources {
+    List<IJsonSchemaValidationProperties> enums;
+
+    public String resourceConstant = ApplicationConstants.RESOURCE;
+
+    public CsharpApiResources(CsharpApiResourceBuilder apiResourceBuilder) {
+        super(apiResourceBuilder);
+        enums = new ArrayList<>(OperationStore.enums.values());
```

Comment:
This can be set where `enums` is defined and then this constructor can be removed.